jupyter_dashboards/nbextension/notebook/dashboard-view/layout/report/layout.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+define([
+    'jquery',
+    '../../../link-css',
+    '../../dashboard-metadata',
+    '../../notebook-util',
+    'template!./cell-controls.html',
+    'template!./collapse-button.html'
+], function(
+    $,
+    linkCSS,
+    Metadata,
+    nbUtil,
+    $cellControlsTemplate,
+    $collapseButtonTemplate
+) {
+    'use strict';
+
+    var cssLoaded = false;
+    var $CONTAINER = $('#notebook-container');
+    var SCROLL_CONTAINER = $('#site').get(0);
```

Comment:
These will be the same across all layouts, so they should probably be passed in as options to the constructors.